client/app/Core/MultiSelectDropDown/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Created by saubhagya on 1/8/17.
+ */
+
+import React, { Component } from 'react';
+
+class MultiSelectDropdown extends Component{
+    constructor(){
+        super();
+        this.state = {
+            selectedVal: [],
+            dataSelected : false,
+            nameVal:'Select'
+        }
+    }
+
+    onSelectedVal = (event) => {
+        let newVal = event.target.value;
+        this.setState({
```

Comment:
Extra code of line is written. Optimize the code lines and remove those are not needed